library/core/src/main/java/com/google/android/exoplayer2/text/subrip/SubripDecoder.java

```diff
@@ -95,8 +128,36 @@ protected SubripSubtitle decode(byte[] bytes, int length, boolean reset) {
         textBuilder.append(currentLine.trim());
       }
 
-      Spanned text = Html.fromHtml(textBuilder.toString());
-      cues.add(new Cue(text));
+      // Extract tags
+      SubtitleTagResult tagResult = extractTags(textBuilder);
+      Spanned text = Html.fromHtml(tagResult.cue);
+
+      Cue cue = null;
+
+      // Check if tags are present
+      if (tagResult.tags.length > 0) {
+
+        boolean alignTagFound = false;
```

Comment:
Can't you just `break;` from the loop when you find the first align tag, and avoid the need for this?

Reply:
Sure I could, but I intended to wrote like this to be generic and to be able to extend. Note that this for loop isn't only for align tags (for now it is), instead goes over all the tags and applies them, I suppose it will have support for more tags in the future and it that case breaking after the first align tag would be incorrect behavior.

Taking these into consideration, if you still would like to just "break", then I can change it.

Reply:
I think our preference (for small blocks of code; this doesn't really apply to larger scale design choices for obvious reasons) is to write it in the simplest way for current usage, and change it as and when the usage changes. So I think using break is still preferable for now.